filebrowser/namers.py

```diff
@@ -2,7 +2,10 @@
 import re
 from django.utils import six
 from django.utils.module_loading import import_string
-from django.utils.text import force_text
+try:
```

Comment:
This should be: `from django.utils.encoding import force_text` (that's compatible with all Django versions and the other changes in this file are unnecessary). The `utils.text` location was a transitive import.